containers-tests/tests/intmap-properties.hs

```diff
@@ -1469,6 +1472,26 @@ prop_partition p ys = length ys > 0 ==>
       m === let (a,b) = (List.partition (apply p . snd) xs)
             in (fromList a, fromList b)
 
+prop_takeWhileAntitone :: Int -> [(Int, Int)] -> Property
+prop_takeWhileAntitone x ys =
+  let l = takeWhileAntitone (<x) (fromList ys)
+  in  valid l .&&.
+      l === fromList (List.filter ((<x) . fst) ys)
```

Comment:
Why `List.filter` and not `List.takeWhile`?

Reply:
It's not a sorted list.

Reply:
Oh yeah....